src/Entity/Entity.h

```diff
@@ -0,0 +1,37 @@
+//
+// Created by piotr on 13.04.18.
+//
+
+
+#ifndef NONAME_ENTITY_H
+#define NONAME_ENTITY_H
+
+#include <SFML/Graphics.hpp>
+#include "../utils/json.hpp"
+
+
+class Entity {
+
+    /*
+     * Skupic sie na NPC, cos ala Prison Architect, zatrudnie gornikow, komunikacja
+     * mapa ID do
+     * */
+
+public:
+    Entity(int ID, std::string name, sf::Vector2f position, sf::RectangleShape hitbox, sf::Sprite sprite);
```

Comment:
Also, this. Good enough for now, but in the future it would be good to use a proper [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern) here to construct objects like that:
```
Entity monster = Entity::new()
    .id(0)
    .name("Monster")
    .position({2.0, 1.0})
    .sprite(someSprite)
    .build();
```
This way we don't have to remember exact type and order of all parameters.